mozdownload/scraper.py

```diff
@@ -6,20 +6,20 @@
 
 from __future__ import absolute_import, unicode_literals
 
+import hashlib
+import io
```

Comment:
As best squash this commit with the previous one which introduced those new imports. Then we can save us from an additional one.